VultisigApp/VultisigApp/Views/Vault/CustomTokenView.swift

```diff
@@ -210,7 +210,8 @@ struct CustomTokenView: View {
                 
                 try await Task.sleep(nanoseconds: 1_000_000_000)
                 isLoading = false
-                chainDetailView.sheetType = nil
+                self.chainDetailView.sheetType = nil
+                dismiss()
```

Comment:
I am not sure why this will cause a crash , and also why this will solve the issue, can you please explain?

Reply:
It is not a crash. It is not dismissing the window, so it doesn’t come back.